modules/fedimint-dummy/src/lib.rs

```diff
@@ -211,8 +212,8 @@ impl ServerModule for Dummy {
     async fn consensus_proposal(
         &self,
         _dbtx: &mut DatabaseTransaction<'_>,
-    ) -> Vec<DummyConsensusItem> {
-        vec![]
+    ) -> ConsensusProposal<DummyConsensusItem> {
+        ConsensusProposal::empty()
```

Comment:
nit: making `await_consensus_proposal` dependent on `consensus_proposal` in the dummy module would show developers using the dummy module as a reference how it should be done.